compiler/src/dotty/tools/dotc/parsing/Scanners.scala

```diff
@@ -48,6 +48,7 @@ object Cbufs {
     def isEmpty: Boolean = buf.length() == 0
     def length: Int = buf.length()
     def last: Char = buf.charAt(buf.length() - 1)
+  end extension
```

Comment:
Can we avoid the `given` line in other places, using the same solution here?

```Scala
given ListOfListDecorator as AnyRef:
```

Reply:
In general, the named ones needed to be in a given or they where not found in other classes.